qa/L0_ensemble_model/check_results.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+# Copyright 2021-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+
+import yaml
+
+
+class TestOutputValidator:
+    """
+    Functions that validate the output
+    of the test
+    """
+
+    def __init__(self, config, test_name, analyzer_log):
+        self._config = config
+        self._models = config["profile_models"]
+        self._analyzer_log = analyzer_log
+
+        check_function = self.__getattribute__(f"check_{test_name}")
+
+        if check_function():
+            sys.exit(0)
+        else:
+            sys.exit(1)
+
+    def check_profile_logs(self):
+        """
+        Check that each model was profiled the number of times
+        corresponding with batch size and concurrency combinations
+
+        (No model config parameter combos expected here!)
+        """
+
+        with open(self._analyzer_log, "r") as f:
+            log_contents = f.read()
+
+        expected_min_num_measurements = 20
```

Comment:
When you ran this on average how many measurements are taken?

Reply:
I observed measurements ranging from 30 to 45 during the test runs. I considered the minimum and maximum counts mentioned above from the [L0_quick_search](https://github.com/triton-inference-server/model_analyzer/tree/main/qa/L0_quick_search) test.